superset-frontend/packages/superset-ui-core/src/theme/GlobalStyles.tsx

```diff
@@ -16,6 +16,18 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+
+// @fontsource/* v5.1+ doesn't play nice with eslint-import plugin v2.31+
+/* eslint-disable import/extensions */
+import '@fontsource/inter/200.css';
```

Comment:
These fonts got carried over from the previous implementation, and they do look nice, but I'm wondering whether they're kind of just bloating the build in some ways. I'm wondering if we could either:
- drop them to and use stock fonts in default build
- dogfood the custom font system in default configs

Probably beyond the scope of this PR, but wanted to bring it up.

Reply:
@kasiazjc what do you think? Could we find some of the best-looking browser stock fonts for vanilla Superset?

Reply:
Inter is nice because not only is it a pretty font, but it includes a lot of OpenType featues, e.g. fixed-spacing for numerals, and a bunch of other goodies.

Reply:
https://rsms.me/inter/